python/aether_routines.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+
+from netCDF4 import Dataset
+import datetime as dt
+import numpy as np
+
+def read_aether_header(filelist):
+    r""" Grab ancillary information from the netcdf file
+
+    Get some ancillary information from the netcdf file.  This simply
+    mirrors the gitm header info.
+
+    Parameters
+    ----------
+    filelist: a list of netcdf files. Really, only the last file is used.
+              also, the number of files is recorded.
+
+    Returns
+    -------
+    header: A dictionary containing information about the netcdf file, such
+            as nLons, nLons, nAlts, nVars, variable names, time(s)
+    """
+
+    header = {"nFiles": len(filelist), \
+              "version": 0.1, \
+              "nLons": 0, \
+              "nLats": 0, \
+              "nAlts": 0, \
+              "nVars": 0, \
+              "vars": [], \
+              "time": [], \
+              "filename": [filelist[-1]] }
+    
+    ncfile = Dataset(filelist[-1], 'r')
+    header["nVars"] = 0
+    for var in ncfile.variables.values():
+        s = var.shape
+        if (len(s) == 3):
+            header["nLons"] = s[0]
+            header["nLats"] = s[1]
+            header["nAlts"] = s[2]
+            header["vars"].append(var.name)
+            header["nVars"] += 1
+
+    base = dt.datetime(1965, 1, 1, 0, 0, 0)
+    time = np.array(ncfile.variables['Time'])
+    current = base + dt.timedelta(seconds = time[0])
+    header["time"].append(current)
+            
+    ncfile.close()
```

Comment:
```suggestion
    with Dataset(filelist[-1], 'r') as ncfile:
        header["nVars"] = 0
        for var in ncfile.variables.values():
            if (len(var.shape) == 3):
                header["nLons"] = var.shape[0]
                header["nLats"] = var.shape[1]
                header["nAlts"] = var.shape[2]
                header["vars"].append(var.name)
                header["nVars"] += 1

        base = dt.datetime(1965, 1, 1, 0, 0, 0)
        time = np.array(ncfile.variables['Time'])
        current = base + dt.timedelta(seconds=time[0])
        header["time"].append(current)
```

Implemented `with`.  Also removed definition of unnecessary variable.
The lack of whitespace for kwargs is useful for visually separating variable equalities from function or method kwarg input.

Reply:
Question: is `time` ever going to be a list of more than one item?  I can't see how :(